HyperVExtension/src/HyperVExtension/HyperVExtension.csproj

```diff
@@ -54,6 +54,10 @@
     <PackageReference Include="Microsoft.Extensions.Http" Version="8.0.0" />
   </ItemGroup>
 
+  <ItemGroup>

```

Comment:
If all resw files need to be included this way, please add a step in https://github.com/microsoft/devhome/blob/main/docs/tools/creating-a-tool.md after the step explaining how to create the resw file.

Reply:
Can this be in a props file?

Reply:
Looks like this was unnecessary. I'm re-running the pipeline to be sure but thanks for pointing this out.